### PR TITLE
rustfmt clippy

### LINKED ISSUE #342: rustfmt clippy
Let's try and run rustfmt on clippy.

Considering that I clippy'd rustfmt some time ago and my PR was accepted, it's only fair that we return the favor :smile:

### PR DESCRIPTION
**Pull request by llogiq**
So I started running rustfmt on our code, which promptly panicked. ;-) Feeding it the submodule source files worked ok-ish though, so that's what I did for now.

The results are...mixed. Perhaps we don't want to merge this yet, or ever. But I wanted to have a data point for the discussion.

This closes #342.

### CODE REVIEW

File: src/bit_mask.rs [hunk 1]
```diff
@@ -79,7 +86,7 @@ fn invert_cmp(cmp : BinOp_) -> BinOp_ {
         BiGt => BiLt,
         BiLe => BiGe,
         BiGe => BiLe,
-        _ => BiOr // Dummy
```
Comment (Manishearth) [inline src/bit_mask.rs]: gah!

Comment (llogiq) [inline src/bit_mask.rs]: gah! indeed.

**Comment by llogiq** [comment]
Since you seem to be ok with merging it, @Manishearth r?

File: src/consts.rs [hunk 2]
```diff
@@ -124,38 +129,41 @@ impl PartialEq for Constant {
...
             (&ConstantRepeat(ref lv, ref ls), &ConstantRepeat(ref rv, ref rs)) =>
                 match lv.partial_cmp(rv) {
-                    Some(Equal) => Some(ls.cmp(rs)),
-                    x => x,
-                },
+                Some(Equal) => Some(ls.cmp(rs)),
+                x => x,
+            },
             (&ConstantTuple(ref l), &ConstantTuple(ref r)) => l.partial_cmp(r),
-             _ => None, //TODO: Are there any useful inter-type orderings?
```
Comment (Manishearth) [inline src/consts.rs]: comment went away

File: src/consts.rs [hunk 3]
```diff
@@ -196,30 +206,40 @@ fn constant_not(o: Constant) -> Option<Constant> {
...
                         UintTy::TyU32 => ::std::u32::MAX as u64,
                         UintTy::TyU64 => ::std::u64::MAX,
-                        UintTy::TyUs => { return None; }  // refuse to guess
+                        UintTy::TyUs => {
+                            return None;
+                        }
                     };
                     (!value & mask, UnsignedIntLit(ity))
                 }
-                UnsuffixedIntLit(_) => { return None; }  // refuse to guess
```
Comment (Manishearth) [inline src/consts.rs]: comment

**Comment by Manishearth** [comment]
I'm not sure I like the function-args-split-multiline thing though. Should we turn that off with a rustfmt.toml?

**Comment by marcusklaas** [comment]
Some of the issues that came up here have been patched recently, so it may be worthwhile to run the latest rustfmt and see what that produces.

**Comment by llogiq** [comment]
Will do, once I get to my PC (I'm on mobile currently). Thanks for the heads-up.

**Comment by llogiq** [comment]
I'm closing this in favor of @marcusklaas' PR.

**Pull request closed without merging**